### ISSUE TITLE
Derp field removed from clientConnectivity block in GET tailnet/example.com/devices response

### ISSUE THREAD

**Issue by heretic098** [frenck/python-tailscale#427]
# Problem/Motivation

As of https://github.com/tailscale/tailscale/commit/b7988b382584d7129b15da7590280f5400123891, the structure of the API response has changed so that the derp field is no longer always returned in the clientConnectivity block. 

## Expected behavior

I expect that the response from

curl -X GET  "https://api.tailscale.com/api/v2/tailnet/example.com/devices?fields=all" -u "ts_api_key:" will be parsed and return a list of devices

## Actual behavior
My script returns a lot of similar errors to:
```
devices -> 8498631107582792 -> clientConnectivity -> derp
  field required (type=value_error.missing)
```

## Steps to reproduce
The following snippet will reproduce the behaviour:

```
async def ts_devices(ts_key, tailnet="example.com"):
    async with Tailscale(
        tailnet=tailnet,
        api_key=ts_key,
    ) as tailscale:

        devices = await tailscale.devices()
        return devices

ts_nodes = asyncio.run(ts_devices(ts_key=ts_key))
```

## Proposed changes

Short term, changing the `src/tailscale/models.py:25` to:
```
derp: Optional[str]
```
Fixes this for me on my local system. However I'm not immediately sure if it should just be removed entirely.

**Comment by heretic098** [comment]
Having read through the tailscale commit a bit more, I think removing the line completely is the better solution. It does work on my local system when I remove it.

**Comment by jhf2442** [comment]
Also confirmed to work when removing the derp line + restart of HA

**Comment by frenck** [comment]
fixed in #428
